src/styles/_catalog.scss

```diff
@@ -14,6 +15,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transform-origin: center;
+    transition-duration: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated